apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -165,10 +170,32 @@ type ProcessCollectionFeatureConfig struct {
 // ContainerCollectionFeatureConfig contains Container Collection configuration.
 // Container Collection is run in the Process Agent.
 type ContainerCollectionFeatureConfig struct {
-	// Enabled enables Process monitoring.
+	// Enables container collection for the Live Container View.
 	// Default: true
 	// +optional
-	Enabled *bool `json:"enabled,omitempty"`
+	LiveContainer *bool `json:"liveContainerEnabled,omitempty"`
+
+	// Enables the OOMKill eBPF-based check.
+	// Default: false
+	// +optional
+	OOMKillCheck *bool `json:"oomKillCheck,omitempty"`
+
+	// Enables the TCP queue length eBPF-based check.
+	// Default: false
+	// +optional
+	TCPQueueLengthCheck *bool `json:"tcpQueueLengthCheck,omitempty"`
```

Comment:
what do you think about having a Features.CoreChecks for items like this?

Reply:
I thought about it and I thought it would become a bit of bag of everything, so I tried to put things closest to a "functional" feature (like container monitoring), though I agree it's not super great either and I'm open to suggestions.

Reply:
ok thanks. though admittedly confusing, this section is for a feature owned by the process side; so similarly we should probably find a different home for the items below..